yarn.lock

```diff
@@ -290,9 +290,9 @@ aggregate-error@^3.0.0:
     clean-stack "^2.0.0"
     indent-string "^4.0.0"
 
-ajv@^6.10.0, ajv@^6.10.2:
+ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.3:
```

Comment:
Hm, this is a bit unexpected. Why did it change?

Reply:
Great question - this is because we added the coveralls package in master but forgot to regen lock. Sent out https://github.com/stripe/stripe-node/pull/1531 to re-generate, and then I'll re-fresh this PR once that is merged to get the history aligned.

Reply:
Done - ptal @pakrym-stripe + restamp